examples/p4est_2d_dgsem/elixir_advection_restart.jl

```diff
@@ -26,7 +27,7 @@ dt = load_dt(restart_filename)
 ode = semidiscretize(semi, tspan, restart_filename)
 
 # Do not overwrite the initial snapshot written by elixir_advection_extended.jl.
-save_solution.condition.save_initial_solution = false
+@reset save_solution.condition.save_initial_solution = false
```

Comment:
This changes behavior documented in our examples, so it is a breaking change. Please add a NEWS.md entry.

Reply:
I mentioned explicitly this and that some other structs have also been made immutable.